frontend/catalyst/pennylane_extensions.py

```diff
@@ -2003,6 +2004,22 @@ def circuit():
             m = measure(0, postselect=1)
             return qml.expval(qml.PauliZ(0))
 
+    >>> circuit()
+    -1.0
+
+    **Example with reset**
+
+    .. code-block:: python
+
+        dev = qml.device("lightning.qubit", wires=1)
+
+        @qjit
+        @qml.qnode(dev)
+        def circuit():
+            qml.Hadamard(0)
+            m = measure(0, reset=True)
+            return qml.expval(qml.PauliZ(0))
+
     >>> circuit()
     -1.0
```

Comment:
I think this should now be 1 right since the qubit is reset?
```suggestion
    1.0
```

Reply:
Done